<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LeafTrackerGUI</class>
 <widget class="QMainWindow" name="LeafTrackerGUI">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>254</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Leaf Tracker GUI</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../brl_logo.gif</normaloff>../brl_logo.gif</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>The top level directory containing all the sessions</string>
    </property>
    <property name="text">
     <string>Root Directory</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>The first session to process</string>
    </property>
    <property name="text">
     <string>Save Directory</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>The last session to process</string>
    </property>
    <property name="text">
     <string>Span</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtRootDir">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>36</y>
      <width>351</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>The top level directory containing all the sessions</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtSaveDir">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>351</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>The location of the masks created by MaskGenGUI</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSaveSel">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>69</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnProcess">
    <property name="geometry">
     <rect>
      <x>452</x>
      <y>190</y>
      <width>70</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Process</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtSpan">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>The height of the camera above the object</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>The last session to process</string>
    </property>
    <property name="text">
     <string>Frame memory</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtMemory">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>130</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>The height of the camera above the object</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>100</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>The last session to process</string>
    </property>
    <property name="text">
     <string>Search radius</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtMaxDisplacement">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>100</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>The height of the camera above the object</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>130</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>The last session to process</string>
    </property>
    <property name="text">
     <string>Filter</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtMinFrames">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>130</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>The height of the camera above the object</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbUsePKL">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Use Adaptive light source vectors to generate results which take into account&lt;br/&gt;the pixel position in the field of view and adjust the light source vector accordingly.&lt;br/&gt;Select the location of the AdaptiveLSV.npz generated by the Generate Adaptive LSV tool&lt;br/&gt;in the box below&lt;br/&gt;If this is unchecked standard PS will be used.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>PKL available?</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtPKLFile">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>160</y>
      <width>351</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Select the location of the AdaptiveLSV.npz file</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPKLFileSel">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>159</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>511</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Data settings</string>
    </property>
    <widget class="QPushButton" name="btnRootSel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>24</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_4</zorder>
   <zorder>edtRootDir</zorder>
   <zorder>edtSaveDir</zorder>
   <zorder>btnSaveSel</zorder>
   <zorder>btnProcess</zorder>
   <zorder>edtSpan</zorder>
   <zorder>label_11</zorder>
   <zorder>edtMemory</zorder>
   <zorder>label_12</zorder>
   <zorder>edtMaxDisplacement</zorder>
   <zorder>label_13</zorder>
   <zorder>edtMinFrames</zorder>
   <zorder>cbUsePKL</zorder>
   <zorder>edtPKLFile</zorder>
   <zorder>btnPKLFileSel</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>edtRootDir</tabstop>
  <tabstop>btnRootSel</tabstop>
  <tabstop>edtSaveDir</tabstop>
  <tabstop>btnSaveSel</tabstop>
  <tabstop>btnProcess</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
